capa/rules/__init__.py

```diff
@@ -1401,154 +1423,320 @@ def __getitem__(self, rulename):
     def __contains__(self, rulename):
         return rulename in self.rules
 
+    # unstable
     @staticmethod
-    def _index_rules_by_feature(rules) -> Tuple[Dict[Feature, Set[str]], List[str]]:
+    def _score_feature(scores_by_rule: Dict[str, int], node: capa.features.common.Feature) -> int:
+        """
+        Score the given feature by how "uncommon" it is, where a higher score is more uncommon.
```

Comment:
I may have missed this but how is "uncommon" defined? e.g. uncommon as in the number of times they appear in a rule? Or uncommon as in how often we expect capa to emit them from a sample?

Reply:
Totally driven by my intuition and not science. Intended to be prevalence in features emitted by capa across a wide range of samples. 

"uncommon" may not be precisely correct but I hope it captured the idea succinctly. 

Thoughts?

Reply:
I think it makes sense but we should add the clarification "prevalence in features emitted by capa across a wide range of samples"

Reply:
resolved in [457cfe0](https://github.com/mandiant/capa/pull/2080/commits/457cfe018053f224af9ad6681e767aee1268fa6c)